src/fprime_gds/common/sequence/README.md

```diff
@@ -0,0 +1,86 @@
+# FPy Advanced Sequencing Language Version 0.1
```

Comment:
Eventually we should put this with F´.  We typically try to avoid directory-READMEs.

Reply:
Happy to move it. Where/when should we move it?